packages/scene-composer/src/components/three-fiber/MotionIndicatorComponent/__tests__/__snapshots__/MotionIndicatorComponentSnap.spec.tsx.snap

```diff
@@ -89,7 +89,7 @@ exports[`MotionIndicatorComponent should render with correct speed and values fr
     <div
       data-testid="linear-plane"
     >
-      [{"speed":3,"foregroundColor":0,"backgroundColor":0,"config":{"numOfRepeatInY":2,"defaultForegroundColor":"red","defaultBackgroundColor":"blue","defaultSpeed":6,"backgroundColorOpacity":1},"scale":[1,2,3]},{}]
+      [{"speed":0,"foregroundColor":0,"backgroundColor":0,"config":{"numOfRepeatInY":2,"defaultForegroundColor":"red","defaultBackgroundColor":"blue","defaultSpeed":6,"backgroundColorOpacity":1},"scale":[1,2,3]},{}]
```

Comment:
why is this changed? something seems to be wrong

Reply:
I can run them again.check what changed this.